package.json

```diff
@@ -9,10 +9,7 @@
     "url": "https://github.com/codecov/sourcegraph-codecov.git"
   },
   "categories": [
-    "Reports and stats",
-    "External services",
```

Comment:
Shouldn't the extension also show up for a search for "External services"? The "Insights" category is also used to find extensions providing insights